projects/observability/src/pages/explorer/explorer.component.ts

```diff
@@ -339,5 +339,6 @@ export const enum ExplorerQueryParam {
   GroupLimit = 'limit',
   Series = 'series',
   Order = 'order',
-  Filters = 'filter'
+  Filters = 'filter',
+  Time = 'time'
```

Comment:
I'm not sure I'd call this an explorer param. Consider treating time separately and always going through TR service.

Reply:
Makes sense, reverted